hackathon_site/dashboard/frontend/src/components/general/ProductOverview/ProductOverview.module.scss

```diff
@@ -0,0 +1,64 @@
+@import "./../../../assets/abstracts/mixins";
+@import "./../../../assets/abstracts/variables";
+
+.mainsection {
+    display: flex;
+    margin: 108px 50px 0 50px;
+    .title {
+        display: flex;
+        font-weight: 500;
+    }
+    .available {
+        margin: 9px 151px 0 0;
+        color: color(grey);
+    }
+    .notAvailable {
+        margin: 9px 151px 0 0;
+        color: red;
+    }
+    .category {
+        margin-top: 59px;
+        font-weight: 500;
+        &Item {
+            color: color(primary);
+            background-color: color(blueLight);
+            margin: 8px 2.5px;
+        }
+    }
+    & img {
+        margin: 0 11px;
+    }
+}
+
+.detailinfosection {
```

Comment:
Minor, but give some level of distinction between the words like `.detailInfoSection`. Same for some of the other class names in this file